docs/core_docs/docs/integrations/document_loaders/file_loaders/oracleai.mdx

```diff
@@ -0,0 +1,79 @@
+# Oracle AI Vector Search: Document Processing
+
+## Load Documents
+
+Users have the flexibility to load documents from either the Oracle Database, a file system, or both, by appropriately configuring the loader parameters. For comprehensive details on these parameters, please consult the [Oracle AI Vector Search Guide](https://docs.oracle.com/en/database/oracle/oracle-database/23/arpls/dbms_vector_chain1.html#GUID-73397E89-92FB-48ED-94BB-1AD960C4EA1F).
```

Comment:
Can you change to say "You" instead of "Users"?  Ditto for other places in the doc.